receiver/k8sclusterreceiver/README.md

```diff
@@ -265,4 +269,29 @@ spec:
           configMap:
             name: otelcontribcol
 EOF
+```
+
+### OpenShift
+
+You can enable OpenShift support to collect OpenShift specific metrics in addition to the default
+k8s ones. To do this, set the OpenShift key in config to `true` to enable OpenShift support.
+
+Example:
+
+```yaml
+  k8s_cluster:
+    openshift: true
```

Comment:
What about maybe making the distribution configurable?

```yaml
k8s_cluster:
  distribution: openshift
```

There might be other features added for other distros.

Reply:
Sounds good.